<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunConfigDlg</class>
 <widget class="QDialog" name="RunConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RunConfigDlg</string>
  </property>
  <widget class="QGroupBox" name="GrpBox_General">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>701</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>General</string>
   </property>
   <widget class="QLabel" name="lb_Start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnEdt_Start">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnEdt_End">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_End">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>End</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_Passes">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Passes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnEdt_Passes">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpBox_Pipeline">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>701</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Pipeline</string>
   </property>
   <widget class="QCheckBox" name="chkBox_Init">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialization</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkBox_BO">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bundle Optimization</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkBox_DE">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Depth Level Expansion</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>671</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hLayout_1">
    <item>
     <widget class="QCheckBox" name="chkBox_OutputTmpDat">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Output Temporary Data</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>259</y>
     <width>671</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="hSpace_Left">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pBtn_OK">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBtn_Cancle">
      <property name="text">
       <string>Cancle</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="hSpace_Right">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
